### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by thepatrick**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/thepatrick/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
```
Comment (thepatrick) [inline ReadMe.md]: This is a comment

Comment (thepatrick) [inline ReadMe.md]: This needs a !fix

Comment (thepatrick) [inline ReadMe.md]: Thanks me  !fixed

**Pull request merged**